guides/release/routing/query-params.md

```diff
@@ -51,11 +51,12 @@ array that the `articles` template will render:
 import Controller from '@ember/controller';
 import { computed } from '@ember/object';
 
-export default Controller.extend({
-  queryParams: ['category'],
-  category: null,
+export default class ArticlesController extends Controller {
```

Comment:
Are we sure that none of these APIs attempt to look at `ControllerClass.prototype` internally?

Reply:
I am not sure what you mean. If they did, how would it change this example?